### PR TITLE
export styled HTML with heading id's

### PR DESCRIPTION
**Pull request by fxha**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| Fixed tickets    | #435
| License          | MIT

### CODE REVIEW

File: src/muya/lib/utils/exportStyledHTML.js [hunk 1]
```diff
@@ -162,6 +165,15 @@ class ExportHTML {
     $(`.${CLASS_OR_ID['AG_ACTIVE']}`).removeClass(CLASS_OR_ID['AG_ACTIVE'])
     $(`[data-role=hr]`).replaceWith('<hr>')
 
+    // create heading id's
+    const headings = $(headingClassNames.join(', '))
+    if (headings.length) {
+      headings.each((i, h) => {
+        const heading = $(h)
+        heading.attr('id', `${heading.attr('id')} ${heading.attr('data-id')}`)
```
Comment (Jocs) [inline src/muya/lib/utils/exportStyledHTML.js]: Is there a space between `${...}` and `${...}`?
There should be no space in `id` attribute according to the standard https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

**Comment by Jocs** [comment]
@fxha Please mark the issues related to the PR I have merged as `fixed but not released`, thanks.

File: src/muya/lib/utils/exportStyledHTML.js (hunk 1 above)
Comment (fxha) [reply src/muya/lib/utils/exportStyledHTML.js]: Yes that's right, but I merge two separate IDs together. E.g. consider `id="ag-0"` and `data-id="foo-bar"` then the result should be `id="ag-0 foo-bar"`. The element now has two IDs `ag-0` and `foo-bar`.

Comment (Jocs) [reply src/muya/lib/utils/exportStyledHTML.js]: I found description of attribute id as bellow:

>This attribute's value must not contain whitespace (spaces, tabs etc.). Browsers treat non-conforming IDs that contain whitespace as if the whitespace is part of the ID. In contrast to the class attribute, which allows space-separated values, **elements can only have one single ID**.

Comment (fxha) [reply src/muya/lib/utils/exportStyledHTML.js]: OK then I'll overwrite the ID attribute with `data-id`.

**Pull request merged**